.circleci/config.yml

```diff
@@ -439,174 +439,174 @@ workflows:
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-multi
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-multi-1'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-multi-1
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-mx'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-multi-mx
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-vanilla'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-vanilla
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-isolation'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-isolation
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-worker'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-worker
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-operations'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-operations
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-follower-cluster'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-follower-cluster
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-columnar'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-columnar
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-columnar-isolation'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-columnar-isolation
           requires: [build-13]
       - tap-test-citus:
           name: 'test_13_tap-recovery'
           pg_major: 13
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           suite: recovery
           requires: [build-13]
       - test-citus:
           name: 'test-13_check-failure'
           pg_major: 13
           image: citus/failtester
-          image_tag: '13.2'
+          image_tag: '13.4-dev202108201324'
           make: check-failure
           requires: [build-13]
 
       - test-pg-upgrade:
           name: 'test-12-13_check-pg-upgrade'
           old_pg_major: 12
           new_pg_major: 13
-          image_tag: 12-13
+          image_tag: '12-13-14-dev202108201324'
           requires: [build-12,build-13]
 
       - test-citus-upgrade:
```

Comment:
Check-merge-to-enterprise is still using `13.2`, should we upgrade that to use 13.4 as well. (I mean it is a minor thing)

Reply:
good one, will update that one aswell. I think it makes sense to move them along at the same time.